### DISCUSSION TITLE
Squares&Circles Patch_SM Prototype

### DISCUSSION THREAD

**Discussion by eh2k** [eh2k/squares-and-circles#24] [General]
Hello,

a little teaser:

Besides the alternate firmware I'm a bit further, and have a new prototype running, which is very similar to the OC. I have deliberately chosen similar components (encoder, display, buttons) because I wanted to make the sourcing as easy as possible for DIYers. Building is actually even easier, because of the central component: the daisy-submodule.

There are several expansion pins ike USB, I2C on the pcb. I am currently unsure if I should replace the Midi-In with a Led-Button (PB6149L). The current pcb is a mixture of through-hole (ICs, Header) and SMD (most resistors - I will try to change them from 603 to 805 in the next PCB version, if there is enough space).

The BOM is straightforward - besides the standard components ( jacks, encoders, buttons) a SH1106 display (same as the OC) and only 2 ICs are needed: 74HC165 shift-register and an opto-coppler (4N35 or H11L1M) for the midi-in.

<img src="https://user-images.githubusercontent.com/11786292/199604413-df9e67ec-d06c-41ec-8157-a3b8d10549f0.jpg" height="300"/>
<img src="https://user-images.githubusercontent.com/11786292/200117675-baf80d86-6cb5-42c8-9e1a-ffdbdc24138d.png" height="300"/>
<img src="https://user-images.githubusercontent.com/11786292/199604075-6a593156-841b-4c04-85b4-e77a4af4fcfe.jpg" height="300"/>

More details: [https://github.com/eh2k/squares-and-cir ... -Prototype](https://github.com/eh2k/squares-and-circles/wiki/Squares&Circles-D_SM-Prototype)

A few considerations regarding software.
In comparison to the OC, we have here only 2 audio outputs. Unfortunately, the DaisySM CV outputs (12bit DAC) designed for the 0-5V range, and not suitable for audio - otherwise I would have used them somehow. However, on the PCB there are 2 jumpers where you can switch A and B jacks between CV or audio in.
There are no direct triggers. The 8 CVs can be used as CV or also as triggers.
On the whole, I have the current "squares-and-circles" firmware running on the DaisySM without any major hurdles. Currently I'm stuck on the ADCs - and try to get out here what is possible - for me that would be a killer feature if I could use all CV inputs also for audio. Compared to Teensy the internal ADCs (16bit) are not so bad.

Feel free for feedback or any hints. I hope to find the time to make the module more "finished". If you have interest in PCB or DIY kit, you could send me a email without obligations - so that I can estimate whether there is any demand at all.

eh2k

**Comment by themoonscientistandco** [comment]
hallo eh2K
nice work ,very interesting .
Especially by using the DaisySM ,we don't need the expensive 8565 dac .
also the teensys are hard to buy at the moment.(in europe )
looking forward for the gerber files .would be happy to make a test module.
603 is ok for me, but for others 805 would probably be more comfortable.
again, many thanks for your fantastic work
grtz
the moonsclientist
